<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OderDetailBar</class>
 <widget class="QWidget" name="OderDetailBar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>821</width>
    <height>51</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="orderTime_lab">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>0</y>
     <width>720</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>订单时间</string>
   </property>
  </widget>
  <widget class="QPushButton" name="delete_ptn">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QPushButton" name="detail_ptn">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>20</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
